.github/workflows/nerdctl.yaml

```diff
@@ -0,0 +1,109 @@
+name: Nerdctl
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'site/**'
+
+permissions:
+  contents: read
+
+jobs:
+  nerdctl:
+    name: Nerdctl
+    runs-on: ubuntu-20.04
```

Comment:
Why not 22.04 ?

Reply:
The kind uses the 20.04 in the other CI, like https://github.com/kubernetes-sigs/kind/blob/v0.20.0/.github/workflows/docker.yaml#L17
So the code is to follow the docker and podman's CI :-)